homeassistant/components/stream/recorder.py

```diff
@@ -169,7 +170,9 @@ def write_transform_matrix_and_rename(video_path: str) -> None:
             os.remove(video_path + ".tmp")
 
         def finish_writing(
-            segments: deque[Segment], output: av.OutputContainer, video_path: str
+            segments: deque[Segment],
+            output: av.container.OutputContainer | None,
+            video_path: str,
```

Comment:
The `output = None` case is already handled explicitly.
https://github.com/home-assistant/core/blob/f139bc14e262b494ba3ae4d54f2c6cb11328fe5b/homeassistant/components/stream/recorder.py#L181-L183